tests/ui/traits/clone-unwind-rc-cleanup.rs

```diff
@@ -1,14 +1,13 @@
+//! Tests `Clone` cleanup behavior during unwinding with `Rc`.
```

Comment:
This test doesn't seem to be intended to test the `Clone` impl of `Rc`, but rather the builtin `Clone` impl for tuples.

Reply:
Yes